src/Core/Utilities/Plan/FreePlan.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Bit.Core.Enums;
+
+namespace Bit.Core.Utilities.Plan;
+
+public record FreePlan: Models.StaticStore.Plan
+{
+    public FreePlan()
```

Comment:
Missing `AllowSeatAutoscale = false`. I guess that'll be its default value anyway, but it's nice to assign it expressly.